From 6a9b0574f80c2a0f0f5144689e59ca456ae9bf90 Mon Sep 17 00:00:00 2001
From: Adri <git@verbto.be>
Date: Mon, 26 Oct 2026 17:46:51 +0100
Subject: [PATCH 1/2] a22x only: drop the iso9660 and udf genfscon rules, the
 stock Samsung vendor labels them vfat and secilc rejects the conflict

Change-Id: Idbfbf82587926b3dcd395adfb6aa8ff80120a7fc
---
 common/private/genfs_contexts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/common/private/genfs_contexts b/common/private/genfs_contexts
index 57f7e0e..efad910 100644
--- a/common/private/genfs_contexts
+++ b/common/private/genfs_contexts
@@ -1,5 +1,3 @@
 genfscon sysfs /devices/virtual/timed_output/vibrator u:object_r:sysfs_vibrator:s0
 genfscon sysfs /module/mmcblk/parameters/perdev_minors u:object_r:sysfs_perdev_minors:s0
 
-genfscon iso9660 / u:object_r:iso9660:s0
-genfscon udf / u:object_r:udf:s0
-- 
2.47.3

